src/components/ConcertCard.js

```diff
@@ -2,71 +2,103 @@ import styled from "styled-components/macro";
 import PropTypes from "prop-types";
 import "../GlobalStyles";
 import "../data.json";
+import Button from "./Button";
 
 ConcertCard.propTypes = {
   concerts: PropTypes.arrayOf(
     PropTypes.shape({
-      concertDate: PropTypes.string,
-      artistName: PropTypes.string,
-      concertLocation: PropTypes.string,
+      date: PropTypes.string,
+      artist: PropTypes.string,
+      location: PropTypes.string,
       id: PropTypes.string,
+      image: PropTypes.node,
+      price: PropTypes.string,
     })
   ),
+  onClick: PropTypes.func,
 };
 
 export default function ConcertCard({
-  concertDate,
-  artistName,
-  concertLocation,
+  date,
+  artist,
+  location,
+  image,
+  onClick,
 }) {
-  const createDate = concertDate.split(", ");
+  const createDate = date.split(", ");
   const Date = [createDate[1].replace(",", " ")];
 
-  const createLocation = concertLocation.split(",");
+  const createLocation = location.split(",");
   const Location = [createLocation[1].replace(",", " ")];
 
   return (
-    <Wrapper>
-      <DateStyle>{Date}</DateStyle>
-      <ArtistStyle>{artistName}</ArtistStyle>
-      <LocationStyle>{Location}</LocationStyle>
+    <Wrapper onClick={onClick}>
+      <TextStyle>
+        <DateStyle>{Date}</DateStyle>
+        <ArtistStyle>{artist}</ArtistStyle>
+        <LocationStyle>{Location}</LocationStyle>
+      </TextStyle>
+      <Image src={image} alt="" />
     </Wrapper>
   );
 }
 
-const Wrapper = styled.li`
+const Wrapper = styled(Button)`
   margin: 3px;
   border: 1px solid white;
-  border-radius: 1em;
+  width: 50vh;
+  border-radius: 10px;
   padding: 10px;
   background: var(--white);
   list-style: none;
   display: grid;
-  grid-template-columns: 1fr 3fr;
-  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 75% 25%;
+  grid-template-rows: 10%;
+  gap: 0px 0px;
```

Comment:
Gap-default is 0, is this really necessary?

Reply:
thx! I tested some different gap sizes here and left it empty in the end. I will delete it :)